community.md

```diff
@@ -1,13 +1,37 @@
 ---
-title: "Community"
+title: "Community repository"
 ---
 
-## An Open Repository for R Packages
+The R-multiverse Community repository democratizes the distribution and deployment of R package releases.
```

Comment:
is "democratize" too vague? "simplifies"?

Reply:
edited in 9fc7d0994d4aed5050b5ca18d7ea4ac4e0799f9c